agent.py

```diff
@@ -21,9 +21,18 @@ def __init__(self, args, env):
     self.discount = args.discount
 
     self.online_net = DQN(args, self.action_space).to(device=args.device)
+    # load model (raise exception on incorrect model paths)
     if args.model and os.path.isfile(args.model):
       # Always load tensors onto CPU by default, will shift to GPU if necessary
       self.online_net.load_state_dict(torch.load(args.model, map_location='cpu'))
+      print("Loading pretrained model: " + args.model)
+
+    elif args.model and not os.path.isfile(args.model):
+      raise FileNotFoundError(args.model)
+
+    else:
```

Comment:
I would delete the `else` branch as this is the default and most common behaviour, so there should only be a need to signpost a trained model.

Reply:
Done